### ISSUE TITLE
RadListView staggered layout not working on iOS using nativescript-vue

### ISSUE THREAD

**Issue by dobomode** [telerik/nativescript-ui-feedback#931]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
Yes

### Tell us about the problem
RadListView staggered layout is not working properly for me on iOS when using nativescript-vue. It seems to be working OK on Android. See screenshot below.

![image](https://user-images.githubusercontent.com/8964150/48473285-9368b100-e811-11e8-89f2-66f54d7cd1af.png)

### Which platform(s) does your issue occur on?
iOS

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** "nativescript-ui-listview": "^3.8.0"
- **CLI:** 5.0.0
- **Cross-platform modules:** ~5.0.0
- **Runtime(s):** 5.0.0

### Please tell us how to recreate the issue in as much detail as possible.
See code below

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

**https://play.nativescript.org/?template=play-vue&id=cxERSm**

```
const words = ['One', 'Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten'];

const getRandomString = () => {
    const length = Math.round((Math.random() * 15));
    let result = words[0];
    for (let i = 0; i < length; i++) {
        result += (words[i % words.length] + ' ');
    }
    return result;
};

const getRandomItems = (size) => {
    let items = [];

    for (let i = 0; i < size; i++) {
        items.push({
            name: `Item ${i}`,
            description: getRandomString(),
        });
    }
    return items;
};

export default {
    template: `
<Frame>
  <Page>
    <StackLayout>
      <RadListView for="item in itemList"
                   layout="staggered"
                   :gridSpanCount="3">
        <v-template>
          <StackLayout class="item" orientation="vertical">
            <Label :text="item.name"></Label>
            <Label :text="item.description" textWrap="true"></Label>
          </StackLayout>
        </v-template>
      </RadListView>
    </StackLayout>
  </Page>
  </Frame>
  `,
    data() {
        return {
            itemList: getRandomItems(20),
        };
    },
};
```

**Comment by sundus-y** [comment]
Having similar issue with grid layout too, anyone have any workarounds? 
Thanks

**Comment by msaelices** [comment]
@sundus-y  It was fixed on the code some hours ago. The next release will have the fix :)

**Comment by VladimirAmiorkov** [comment]
This issue is resolved in the **5.1.1**  version of `nativescript-ui-listview`
